src/deploy/functions/prepare.ts

```diff
@@ -87,5 +89,28 @@ export async function prepare(context: any, options: any, payload: any): Promise
   const localFnsInRelease = payload.functions.triggers.filter((fn: CloudFunctionTrigger) => {
     return functionMatchesAnyGroup(fn.name, context.filters);
   });
+
+  const res = await gcp.cloudfunctions.listAllFunctions(context.projectId);
+  if (res.unreachable) {
+    const regionsInDeployment = Object.keys(payload.functions.byRegion);
+    const unreachableRegionsInDeployment = regionsInDeployment.filter((region) =>
+      res.unreachable.includes(region)
+    );
+    if (unreachableRegionsInDeployment) {
+      throw new FirebaseError(
+        "The following Cloud Functions regions are currently unreachable:\n" +
+          unreachableRegionsInDeployment.join("\n") +
```

Comment:
nit: Wouldn't it be more consistent in the style to put a tab before elements in a list?